### PR TITLE
Hotfix/maptool 2.0.2

### PR DESCRIPTION
**Pull request by dan-du-car**
# PR Details
## Description
TileAge displayed on the MAPTool ISD map is updated every time user zoom in/out, move around a map. This update incurs bing map API billable calls. It is observed that there are more than 1200 calls to this bing map API a day, which leave the team around 100 days before it exhausts all the free API usage with this free version bing map account.   

## Related GitHub Issue
NA

## Related Jira Key
https://usdot-carma.atlassian.net/browse/MAP-222

## How Has This Been Tested?
Local integration test

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by maefromm** [comment]
@dan-du-car  as requested please confirm here if you've done a sweep of the PRs that may have missed in addition to the one for tile age. Thank you!

**Comment by dan-du-car** [comment]
> @dan-du-car as requested please confirm here if you've done a sweep of the PRs that may have missed in addition to the one for tile age. Thank you!

@maefromm I checked the dockerfile, release notes, session key PRs and they look good. I think monitoring service is in good shape since we have no issue with that in webappopen.

**Review by maefromm — CHANGES_REQUESTED** [review]
@dan-du-car 
1) for this  PR #58 I noticed there's no changes to bing.js and there's differences in changes in index and mapping.js. Could you explain why those changes are different from
https://github.com/usdot-fhwa-stol/connectedvcs-tools/pull/39/files
PR? Do you no longer need those changes? 
2) For the sonarcloud, please address the hotspots issues by logging in and hitting review and choosing the action for it.
Thank you!

File: fedgov-cv-ISDcreator-webapp/src/main/webapp/js/mapping.js [hunk 1]
```diff
@@ -823,8 +831,8 @@ function init() {
 	for(var key in controls) {
 		map.addControl(controls[key]);
 	}
-
-    map.events.register("moveend", map, tileAge);
+	//Temporarily commented out
```
[the remark this answers is not in the document]
Comment (dan-du-car) [reply fedgov-cv-ISDcreator-webapp/src/main/webapp/js/mapping.js:834]: We decided that tileage is not a critical and temporarily disable it as we are moving towards new map provider. I stated this change in the hotfix request document. It is not the exact same change we made before.

[the remark this answers is not in the document]
Comment (maefromm) [reply fedgov-cv-ISDcreator-webapp/src/main/webapp/js/mapping.js:834]: @dan-du-car how about these changes, are these also related to tileage rollback? 
image

**Review by maefromm — APPROVED** [review]

**Pull request merged**